service_configuration_lib/spark_config.py

```diff
@@ -357,15 +342,15 @@ def get_dra_configs(spark_opts: Dict[str, str]) -> Dict[str, str]:
     )
 
     min_ratio_executors = None
-    default_dra_min_executor_ratio = default_spark_conf.get('spark.yelp.dra.minExecutorRatio', 0.25)
+    default_dra_min_executor_ratio = default_spark_srv_conf['spark.yelp.dra.minExecutorRatio']
```

Comment:
I am assuming all of these defaults have already been moved to spark.yaml file

Reply:
Yes these have been moved